packages/shared-chat-completion/src/types/chat-completion-options.ts

```diff
@@ -5,6 +5,7 @@ import type { ToolChoice } from './tool-choice'
 
 export interface ChatCompletionOptions extends CommonRequestOptions {
   [key: string]: unknown
+  maxRoundTrip?: number
```

Comment:
The naming should ideally be consistent with Vercel AI SDK, i.e. `maxSteps`.

Also, it's better to put it in `GenerateTextOptions` instead of `ChatCompletionOptions` when other packages (generate-object, stream-text) don't support this option.

Reply:
Agreed.